src/leappto/cli.py

```diff
@@ -756,11 +745,24 @@ def print_migrate_info(text):
 
         # if el7 then use systemd
         if machine_src.installation.os.version.startswith('7'):
-            result = mc.start_container('centos:7',
-                                        '/usr/lib/systemd/systemd --system',
-                                        tcp_mapping)
+            opts = [
+                # Required for docker 1.13
+                "--cap-add", "SYS_ADMIN",
+                # Implemented in default command
+                #"-v", "/sys/fs/cgroup:/sys/fs/cgroup:ro",
+                "--tmpfs", "/run", 
+                "--tmpfs", "/tmp",
+                "-e", "container=docker"
+                
+            ]
+
+            result = mc.start_container('centos/systemd',
```

Comment:
This should have a comment noting that `centos/systemd` doesn't currently offer version labels, so for now we're just relying on "CentOS 7 is the latest version".